src/components/productList/Filter/Checkbox.js

```diff
@@ -0,0 +1,33 @@
+import { Input } from "./Input";
+
+import { useData } from "../../../Context/state-context";
+export const Checkbox = ({ filter }) => {
+  const {
+    state: { selectedFilters },
+  } = useData();
+  let value = selectedFilters?.category?.data;
+
+  console.log("category", value);
+
+  return (
+    <>
+      <div className='card-divider'></div>
+
+      <li className='filter-section-title'>{filter.name}</li>
+      {filter.data.map((check) => (
+        <li key={check.id}>
+          <Input
+            type='CHECKBOX_CHANGE'
+            filterType={filter.type}
+            inputType='checkbox'
+            label={check.label}
```

Comment:
You can provide a better specific name for dispatch type as it was confusing at the first glance. 
Example:
`
 type='CHECKBOX_CHANGE'
`

`
 dispatchType='CHECKBOX_CHANGE'
`